zwave_js_server/firmware.py

```diff
@@ -44,7 +45,7 @@ async def update_firmware(
 
 async def driver_firmware_update_otw(
     url: str,
-    firmware_file: DriverFirmwareUpdateData,
+    firmware_file: FirmwareUpdateData,
```

Comment:
I've renamed this since the driver has all items that the node firmware updates has.

Reply:
I have no plans for this to diverge, as it would require changes to the update service for no significant benefit.

Reply:
I think I should make a wrapper anyway to make the models consistently named